config/config.go

```diff
@@ -111,6 +109,8 @@ type BaseConfig struct {
 
 	DatabaseConnections     int  `mapstructure:"db-connections"`
 	DatabaseLatencyMetering bool `mapstructure:"db-latency-metering"`
+
+	NetworkHRP string `mapstructure:"network-hrp"`
```

Comment:
Do we need the global if the HRP is part of the config? If it is too much effort, I would at least create an issue for this to refactor in the future.

Reply:
the main problem is that hrp is used when address is printed. there are a couple of logs with addresses across the codebase. maybe we can reconsider